### PR TITLE
Header brands ui

### PR DESCRIPTION
**Pull request by jaypal412**
## Why?

Changes in the header menu

## This change addresses the need by:

**Header Bands Menu**

In this, I have added the hover effect to a menu and the alignment of all brands images

**Header Category menu**
- In this, Now I have added the hover effects to the button.
- On scroll button is changing now to the cross icon similar to reference site.
- In this two point are remaining like
    - show the pet image to the background of the drop down.
    - by default open  dog brands in the dropdown

[Delivers #158208438]
[Story](https://www.pivotaltracker.com/story/show/158208438)

### CODE REVIEW

File: src/app/home/content/product-list/product-list.component.html [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
-            </div>
-            <div itemprop="name" class="product-brand">{{ product.name }}</div>
-            <div itemprop="offers" itemscope itemtype="https://schema.org/Offer" class="product-price">
-              <span>
-                <span itemprop="price" class="product-discountedPrice">&#8377; {{ product.price }}</span>
+<div class="container "><div class="row results-base">
+    <div class="col-12 col-md-3 col-sm-3 col-lg-3 product-base" *ngFor="let product of products.products | filter : selectedTaxonIds; trackBy: trackByFn"
+      [style.margin]="getMargin()" itemscope itemtype="https://schema.org/Product">
+
+      <div class="product-thumbShim"></div>
```
Comment (gopalshimpi) [inline src/app/home/content/product-list/product-list.component.html]: Remove this `<div class="product-thumbShim"></div>`
Please format document.(indentation)

File: src/app/home/content/product-list/product-list.component.html [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-      </div>
 
-</div>
-<!-- <div class="product-product" innerHTML="{{ product.description }}"></div> -->
\ No newline at end of file
+        </div>
+      </a>
+    </div>
+  </div>
+  <!-- <div class="product-product" innerHTML="{{ product.description }}"></div> -->
```
Comment (gopalshimpi) [inline src/app/home/content/product-list/product-list.component.html]: Remove commented code.

File: src/app/layout/header/components/categories-menu-dropdown/categories-menu-dropdown.component.html [hunk 3]
```diff
@@ -1,24 +1,23 @@
-<div class="btn-group dpbtn petdropdown" dropdown [autoClose]=autoclose container="body" *ngIf="!dropdownWidth">
-  <button id="button-nested" dropdownToggle categories="button" class="btn dropdown-toggle aba" aria-controls="dropdown-nested"
-    data-hover="dropdown">
-    <i class="fa fa-2x fa-bars" *ngIf="isScrolled"></i>
+<div class="dpmenu">
+  <div  class="" [ngClass]="[isOpen ? 'open main-nav-trigger-open':'',isScrolled ? 'main-nav-trigger':'cust-menu-btn']" (mouseenter)="dropdown.show()" (mouseleave)="dropdown.hide()">
```
Comment (gopalshimpi) [inline src/app/layout/header/components/categories-menu-dropdown/categories-menu-dropdown.component.html]: remove empty lines.

Comment (jaypal412) [reply src/app/layout/header/components/categories-menu-dropdown/categories-menu-dropdown.component.html]: Removed

File: src/app/home/content/product-list/product-list.component.html (hunk 2 above)
Comment (jaypal412) [reply src/app/home/content/product-list/product-list.component.html]: Removed

File: src/app/home/content/product-list/product-list.component.html (hunk 1 above)
Comment (jaypal412) [reply src/app/home/content/product-list/product-list.component.html]: Removed

**Comment by pkrawat1** [comment]
@jaypal412 Please run formatter on all files

File: src/app/layout/header/header.component.scss [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
     z-index: 100;
     width: 100%;
   }
 }
-
+.page-scroll-header{
+  padding-top:0px;
+  padding-bottom:0px;
+}
+:host /deep/ .page-scroll-header .cust-menu-btn{ border-radius:  0px;margin-right: 10px;}
```
[the remark this answers is not in the document]
Comment (jaypal412) [reply src/app/layout/header/header.component.scss]: Done

File: src/app/layout/header/components/categories-menu-dropdown/categories-menu-dropdown.component.ts [hunk 5]
```diff
@@ -83,4 +86,8 @@ export class CategoriesMenuDropdownComponent implements OnInit {
   childCatLoaded(status) {
     this.backBtnShow = status;
   }
```
[the remark this answers is not in the document]
Comment (jaypal412) [reply src/app/layout/header/components/categories-menu-dropdown/categories-menu-dropdown.component.ts]: Done

**Pull request merged**